src/confluent_kafka/schema_registry/schema_registry_client.py

```diff
@@ -686,12 +686,14 @@ class Schema(object):
     Args:
         schema_str (str): String representation of the schema.
 
+        schema_type (str): The schema type: AVRO, PROTOBUF or JSON.
+
         references ([SchemaReference]): SchemaReferences used in this schema.
 
-        schema_type (str): The schema type: AVRO, PROTOBUF or JSON.
-    """
+        named_schemas (dict): Named schemas
```

Comment:
Not added in the args.

Remove this file as there are no changes to this file.

Reply:
Thanks, removed this.